katalog/tests/calico/tigera.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+# Copyright (c) 2022 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+# shellcheck disable=SC2154
+
+load ./../helper
+
+@test "Nodes in Not Ready state" {
+    info
+    nodes_not_ready() {
+        kubectl get nodes --no-headers | awk  '{print $2}' | uniq | grep -q NotReady
+    }
+    run nodes_not_ready
+    [ "$status" -eq 0 ]
+}
+
+@test "Install Prerequisites" {
+    info
+    install() {
+        kubectl apply -f 'https://raw.githubusercontent.com/sighupio/fury-kubernetes-monitoring/v2.0.1/katalog/prometheus-operator/crds/0servicemonitorCustomResourceDefinition.yaml'
+        kubectl apply -f 'https://raw.githubusercontent.com/sighupio/fury-kubernetes-monitoring/v2.0.1/katalog/prometheus-operator/crds/0prometheusruleCustomResourceDefinition.yaml'
```

Comment:
these are a little outdated
```suggestion
        kubectl apply -f 'https://raw.githubusercontent.com/sighupio/fury-kubernetes-monitoring/v3.1.0/katalog/prometheus-operator/crds/0servicemonitorCustomResourceDefinition.yaml'
        kubectl apply -f 'https://raw.githubusercontent.com/sighupio/fury-kubernetes-monitoring/v3.1.0/katalog/prometheus-operator/crds/0prometheusruleCustomResourceDefinition.yaml'
```